src/visitpy/examples/test1.py

```diff
@@ -67,19 +67,19 @@ def AddPlotToWindow(p, v):
     pp = PlotPlugins()
     i = 0
     for p in pp:
-        if(dbs.has_key(p)):
+        if(p in dbs):
```

Comment:
extran parens necessary?

Reply:
no in this case, but still valid (safe output from 2to3)